src/Middleware/NavigationMiddlewareFactory.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+namespace Zend\Navigation\Middleware;
+
+use Interop\Container\ContainerInterface;
+use Zend\Navigation\Navigation;
+
+class NavigationMiddlewareFactory
+{
+    /**
+     * Top-level configuration key indicating navigation configuration
+     *
+     * @var string
```

Comment:
@webimpress It's been this way forever in both phpDocumentor (which predates PSR-5 by 15 years; PSR-5 is largely an attempt to codify what phpDocumentor has been doing all along) and in IDE support; it's the only supported, recognized way of annotating constants. No tools recognize `@const`, unfortunately.